QUICKSTART.md

```diff
@@ -0,0 +1,131 @@
+
+[![Build Status](https://apm-ci.elastic.co/view/All/job/elastic+apm-integration-testing+master+push/badge/icon?style=plastic)](https://apm-ci.elastic.co/job/elastic+apm-integration-testing+master+push/)
+
+## Prerequisites
+
+The basic requirements for starting a local environment are:
+
+- Docker
+- Python (version 3 preferred)
+
+This repo is tested with Python 3 but best effort is made to make starting/stopping environments work with Python 2.7.
+
+### Docker
+
+[Installation instructions](https://www.docker.com/community-edition)
+
+### Python 3
+
+- Windows: [Installation instructions](https://www.python.org/downloads/windows/)
+- Mac (using [Homebrew](https://brew.sh/)):
+  ```sh
+  brew install python
+  ```
+- Debian/Ubuntu
+  ```sh
+  sudo apt-get install python3
+  ```
+
+## Running Local Enviroments
+
+### Starting an Environment
+
+The tool that we use to start and stop the environment is `scripts/compose.py`.  This provides a handy cli for starting an APM environment using docker-compose. 
+
+#### TL;DR
+
+Start an env by running:
+`./scripts/compose.py start --all 6.4`
+
+This will start a complete environment, which includes all of the sample apps and hits them each with a load generator.  Once that is done (and everything has started up) you can navigate to [Your local Kibana Instance](http://localhost:5601/app/apm#/)
+
+#### Details
+
+If you don't want to start everything (for example, on a laptop with limited resources, trying to run zoom at the same time) you can pick and choose which services you run.  Say, for example, that you want to run node, java, and rum.  You could use this command:
+```console
+scripts/compose.py start \
+    --release \
+    --with-opbeans-node \
+    --with-opbeans-rum \
+    --with-opbeans-java
+```
+
+There are many other configuration options, but this is a quickstart.  See the [README](/README.md).
+
+If you want to see what services are available to start, you can run: `./scripts/compose.py start --help | grep "^  --with-opbeans"` which will filter out a list of the agent envs:
+```console
+  --with-opbeans-go     Enable opbeans-go
+  --with-opbeans-java   Enable opbeans-java
+  --with-opbeans-node   Enable opbeans-node
+  --with-opbeans-python
+  --with-opbeans-ruby   Enable opbeans-ruby
+  --with-opbeans-rum    Enable opbeans-rum
+```
+So when new agents are added we don't have to update these instructions.
+
+
+**Bonus**:  With either the `all` or individual methods above, you can also pass a `--with-metricbeat` flag, which will also set up metricbeat.  One side note here is that you will need to then probably need to set a default index pattern
+
+#### Status
+
+Each app gets its own port.  You can actually hit them with your browser.  They all have a similar look & feel.
+
+You can check the status of your APM cluster with `/scripts/compose.py status`, which basically calls :
```

Comment:
either drop the `/` or prefix with `./` - usage is mixed throughout this doc, would be good to sync them all up.

Reply:
Will do.  That one was a copy paste.  Nice catch